src/main/java/org/tc/mail/MailNotificationSender.java

```diff
@@ -42,15 +49,15 @@ public void sendManagerNotification(Course course) {
         try {
             MimeMessageHelper helper = new MimeMessageHelper(message, true);
             helper.setSubject(COURSE_ANNOUNCEMENT_SUBJECT);
-            String[] emailsTo = {roleService.getDepartmentManager().getEmail(),
-                    roleService.getKnowledgeManager().getEmail()};
+            String[] emailsTo = {roleService.getDepartmentManager().get().getEmail(),
+                    roleService.getKnowledgeManager().get().getEmail()};
             helper.setTo(emailsTo);
             helper.setCc(course.getOwner().getEmail());
             helper.setText(contentBuilder.buildManagerNotification(course), true);
+            javaMailSender.send(message);
         } catch (MessagingException e) {
-            LOG.warn("Message didn't send on courseId {0}", course.getId());
+            LOG.debug(String.format("Message didn't send on courseId %1$d", course.getId()));
```

Comment:
One more time - very BAD exception handling.

The log should be WARN level.
And you should log exception so either developer or operations team can understand what the reason of message sending failure.